test/Libraries/RevitServicesTests/RevitServicesTests.cs

```diff
@@ -328,5 +329,34 @@ public void TwoInstancesOfSerializableIdAreEqual()
 
         }
 
+        [Test]
+        [Category("UnitTests")]
+        public void MaterialManagerCheckDefaultValues()
```

Comment:
@mjkkirschner I added two more tests with Revit 2015 projects that covers the part with duplicated material names. No need to run localized Revit. I will merge this soon if you have no objections and then start cherry-picking for 2015 and 2016 later today if everything looks good.